components/timezone/src/ixdtf.rs

```diff
@@ -41,6 +41,23 @@ pub enum ParseError {
     MismatchedTimeZoneFields,
     /// An unknown calendar was provided.
     UnknownCalendar,
+    /// A timezone calculation is required to interpret this string, which is not supported.
```

Comment:
"which is not supported" feels confusing to me, is the specific time zone not supported, is calculating timezones not supported by ICU4X, is calculating timeones not supported by ICU4X in this specific API, or ?

Seems like the last one, if it is the last one then we should suggest what folks should do instead.

Reply:
There's an example suggesting what to do instead.

Reply:
Not really; it suggests changing the string. Someone seeing this error may not have control over the string, we should have solid advice for them when they see this error. Perhaps it is "we don't support `Z` plus a specific timezone"

It feels strange for a timezone calculation library to complain that something requires calculation.

Reply:
Changing the string is the only solution though. We don't support strings of this shape.

> It feels strange for a timezone calculation library to complain that something requires calculation.

### **We are not a timezone calculation library**